content/tutorial-static_equilibrium.md

```diff
@@ -0,0 +1,384 @@
+```
+
+In this tutorial you will use the following NumPy tools:
+
+* [`np.linalg.norm`](https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html) : this function determines the measure of vector magnitude
+* [`np.cross`](https://numpy.org/doc/stable/reference/generated/numpy.cross.html) : this function takes two matrices and produces the cross product
+
++++
+
+# Solving equilibrium with Newton's second law
+
+Your model consists of a beam under a sum of forces and moments. You can start analyzing this system with Newton's second law: 
+
+$$\sum{\text{force}} = \text{mass} * \text{acceleration}.$$
+
+In order to simplify the examples looked at, assume they are static, with acceleration $=0$. Due to our system existing in three dimensions, consider forces being applied in each of these dimensions. This means that you can represent these forces as vectors. You come to the same conclusion for [moments](https://en.wikipedia.org/wiki/Moment_(physics)), which result from forces being applied a certain distance away from an object's center of mass.
+
+Assume that the force $F$ is represented as a three-dimensional vector
+$$F = (F_x, F_y, F_z)$$
+where each of the three components represent the magnitude of the force being applied in each corresponding direction. Assume also that each component in the vector
+$$r = (r_x, r_y, r_z)$$
+is the distance between the point where each component of the force is applied and the centroid of the system. Then, the moment can be computed by
+$r \times F = (r_x, r_y, r_z) \times (F_x, F_y, F_z).$$
+
+Start with some simple examples of force vectors
+
+```{code-cell} ipython3
+forceA = np.array([1, 0, 0])
+forceB = np.array([0, 1, 0])
+print('Force A =', forceA)
+print('Force B =', forceB)
+```
+
+This defines `forceA` as being a vector with magnitude of 1 in the $x$ direction and `forceB` as magnitude 1 in the $y$ direction.
+
+Use matplotlib to visualize these forces.
+
+```{code-cell} ipython3
+fig = plt.figure()
+
+d3 = fig.gca(projection='3d')
+
+d3.set_xlim(-1,1)
+d3.set_ylim(-1,1)
+d3.set_zlim(-1,1)
+
+x, y, z = np.array([0,0,0]) # defining the point of application.  Make it the origin
+
+u, v, w = forceA # breaking the force vector into individual components
+d3.quiver(x,y,z,u,v,w) # plotting forceA
+
+u, v, w = forceB
+d3.quiver(x,y,z,u,v,w) # plotting forceB
+
+plt.show()
+```
+
+There are two forces emanating from a single point.
+
+
+In order to simplify this problem, you can add them together to find the sum of forces.
+
+```{code-cell} ipython3
+forceC = forceA + forceB
+print('Force C =', forceC)
+```
+
+Force C now acts as a single force that represents both A and B.
+You can plot it to see the result.
+
+```{code-cell} ipython3
+fig = plt.figure()
+
+d3 = fig.gca(projection='3d')
+
+d3.set_xlim(-1,1)
+d3.set_ylim(-1,1)
+d3.set_zlim(-1,1)
+
+x, y, z = np.array([0,0,0])
+
+u, v, w = forceC
+d3.quiver(x,y,z,u,v,w)
+
+plt.show()
+```
+
+There is now a single vector at the origin.

```

Comment:
```suggestion
d3.set_xlim(-1,1)
d3.set_ylim(-1,1)
d3.set_zlim(-1,1)

x, y, z = np.array([0,0,0])

u, v, w = forceA
d3.quiver(x,y,z,u,v,w,color='r')
u, v, w = forceB
d3.quiver(x,y,z,u,v,w,color='b')
u, v, w = forceC
d3.quiver(x,y,z,u,v,w,color='g')

plt.show()
```

There is now a larger green vector that is a combination of both force A and force B.
This represents both of them acting at once.
```